source/horoscope/horoscope.html

```diff
@@ -38,6 +38,7 @@
         </select><br>
 
         <br>
+        <input type="button" id="submit" value="SUBMIT">
```

Comment:
I don't think we need it with the new design the submitting would happen from the landing page.